public/admin.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * 
+ */
+
+console.log("Hello to the admin")
+
+var SectionToTagApp = angular.module('SectionToTag', []);
+SectionToTagApp.controller('tagsPickerAppCtrl', function($scope,$http) {
+    $scope.tag_search_results = []
+    $scope.newSearchFragment = function(){
+        $http({
+            method : "GET",
+            url : "https://content.guardianapis.com/tags?api-key="+CONFIG.CAPI_API_KEY+"&q="+encodeURIComponent($scope.searchfragment)
+        }).then(function mySuccess(response) {
+            $scope.tag_search_results = [];
+            angular.forEach(response.data.response.results, function(item, key) {
+                this.push(item.id);
+            }, $scope.tag_search_results);
+        }, function myError(response) {
+            console.log(response.statusText)
```

Comment:
Done. And thanks for pointing this out. It was the result of indiscriminate copy-pasting from the Angular documentation...